### PR TITLE
New recommendation for geo-redundant setups

### PR DESCRIPTION
**Pull request by SkylineCookieMonster**

### CODE REVIEW

**Comment by SkylineCookieMonster** [comment]
@OnyxBit can you review this pull request?
For the Apache Cassandra installer I'll create another pull request as that one requires some extra work and I would like to avoid that this change is blocked because of it.

File: user-guide/Advanced_Functionality/Databases/Supported_system_data_storage_architectures/Dedicated_clustered_storage.md [hunk 1]
```diff
@@ -74,13 +74,11 @@ While hosting Elasticsearch and DataMiner nodes on the same machine is already n
...
 
-Note that **setting up an OpenSearch/Elasticsearch cluster across high-latency nodes is not advised**, as this could compromise proper functionality. This means that it is not possible to simply take OpenSearch/Elasticsearch nodes and spread them out over two or more locations if there is high latency between those locations. This restriction specifically applies to latency between the openSearch/Elasticsearch nodes, not between the DataMiner nodes and the OpenSearch/Elasticsearch nodes. This is why we recommend that instead data is offloaded to multiple OpenSearch clusters.
+The figure below illustrates the recommended minimum setup for geo-redundancy with on-premises machines. For Cassandra, the built-in [NetworkTopologyStrategy](https://cassandra.apache.org/doc/4.0/cassandra/cql/ddl.html#networktopologystrategy) is used in order to configure geo-redundancy. For OpenSearch, we configure one cluster for which the nodes are spread over two data centers and there one tie-breaker in a third site (can be hosted in the cloud).
```
Comment (OnyxBit) [inline user-guide/Advanced_Functionality/Databases/Supported_system_data_storage_architectures/Dedicated_clustered_storage.md:81]: for which the nodes are spread over three zones, two of which host the data whilst the third zone has a single node which does not handle any data and functions as tie-breaker for cluster configuration. The tie-breaker node can be cloud hosted if the latency is sufficiently low (sub 50 ms). For this we leverage [shard allocation awareness](https://opensearch.org/docs/2.12/tuning-your-cluster/#advanced-step-6-configure-shard-allocation-awareness-or-forced-awareness).

**Review by OnyxBit — COMMENTED** [review]
Minor edit for preference on the OpenSearch zone description.
Trying to stick to the zone terminology they use in OpenSearch.

**Comment by SkylineCookieMonster** [comment]
@OnyxBit can you double check if the changes are clear now? Thanks

**Review by OnyxBit — APPROVED** [review]
LGTM on content
Formatting expertise I'll leave to the rulers of that domain. :)

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**